.github/workflows/pipeline.yml

```diff
@@ -0,0 +1,42 @@
+name: CI
+
+on:
+  push:
+
+jobs:
+  tests:
+    runs-on: ubuntu-22.04
+    strategy:
+      matrix:
+        python-version: ["3.8", "3.9", "3.10", "3.11"]
+        pyspark-version: ["3.3.2", "3.4.0"]
+        exclude:
+          - python-version: "3.11"
+            pyspark-version: "3.3.2"
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v4
+        with:
+          python-version: ${{ matrix.python-version }}
+
+      - name: Set up Java
+        uses: actions/setup-java@v3
+        with:
+          java-version: 8
```

Comment:
is it worth running the tests with java 8 and 11?

Reply:
:heavy_check_mark: